patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -191,8 +193,10 @@
          else
          {
 +            harvesters.set(p_180657_2_);
++            harvestTileEntity.set(p_180657_5_);
```

Comment:
This was a dirty hack, and is stupid slow. Better not hold a object and not add two thread locals. I am still not sold on having to pass in the TileEntity itself... it *should* still be accessible in this event.